response_operations_ui/error_handlers.py

```diff
@@ -14,10 +16,22 @@
 def api_error(error):
     logger.error('Api failed to retrieve required data', url=error.url,
                  status_code=str(error.status_code), exc_info=error)
-    return "FAIL"
+    return redirect(url_for('error_bp.server_error_page',
```

Comment:
Should these always be displaying a server error to the user. As Api error is regularly not a 500 server error.

Reply:
I copied this form ras-frontstage.  In that repo an api error has the same page rendered so I wanted to make it consistant.

Reply:
It's a system error but not necessarily in the UI, it's a question that came up in frontstage a lot. 
If one of the external Api's fail do we consider that different from if there's a failure directly in the UI?